server/views/pages/createGoals/index.njk

```diff
@@ -165,7 +165,7 @@
                     Add a note to this goal (optional)
                   </label>
                 </h1>
-                <div class="govuk-warning-text govuk-!-margin-top-4" id="warning-text">
+                <div class="govuk-warning-text govuk-!-margin-top-4">
```

Comment:
Removed the `id` attribute as a) we dont use it in any cypress tests etc so it has little value, and b) it caused accessibility failures when there are multiple goals because this section is in a loop (one for each goal) which meant there were multiple elements on the page with the `id` `warning-text`